doc/api/util.md

```diff
@@ -608,11 +603,9 @@ assert.strict.equal(
 );
 ```
 
-Please note that `util.inspect()` is a synchronous method that is mainly
-intended as a debugging tool. Its maximum output length is limited to
-approximately 128 MB and input values that result in output bigger than that
-will not be inspected fully. Such values can have a significant performance
-overhead that can block the event loop for a significant amount of time.
+`util.inspect()` is a synchronous method for debugging. Its maximum output
```

Comment:
The `mainly intended` part was intentional as it's not entirely limited to debugging. I personally would rather keep that or remove the purpose altogether.

Reply:
@Trott please see the comment above in response to your suggested edits.

Reply:
@BridgeAR How about this?:

```markdown
`util.inspect()` is a synchronous method intended for debugging.
```

By re-inserting "intended", we open the door for there being other uses. But by omitting the softener "mainly", we avoid the appearance of *inviting* other uses.

Also: Yellow! The bike shed MUST be yellow!